### ISSUE TITLE
face_segmentation.pth file

### ISSUE THREAD

**Issue by rikabi89** [Markfryazino/wav2lip-hq#27]
There doesn't appear to a working link to relevant face_segmentation.pth file? Please can update the link.

**Comment by Byronius7X** [comment]
Yeah, the drive link here only linked to the esrgan model. I was able to find the missing one after a search https://drive.google.com/file/d/154JgKpzCPW82qINcVieuPH3fZ2e0P812/view is face_segmentation.pth (just need to rename)
